react-native-purchases-ui/ios/PurchasesPlugin.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  PurchasesPlugin.swift
+//  RNPurchases
+//
+//  Created by César de la Vega  on 10/6/20.
+//  Copyright © 2020 Facebook. All rights reserved.
+//
+
+import Foundation
```

Comment:
Added this empty swift file just in case. We had to add it in the library so xcode configures swift correctly when installing our SDK, which depends on a swift package